docker-compose.yml

```diff
@@ -61,8 +61,7 @@ services:
       - GOOGLE_APPLICATION_CREDENTIALS=/tmp/credentials
     volumes:
       - $PWD:/app
-      # uncomment after setting appropriate .env variables to enable BigQuery testing
-      # - ${GOOGLE_APPLICATION_CREDENTIALS}:/tmp/credentials
+      - /tmp/credentials:/tmp/credentials
```

Comment:
Makes sense to be explicit here.

Reply:
Oops, I actually didn't mean to change this...it breaks localdev by default.